modules/service/src/main/scala/lucuma/odb/graphql/util/MappingExtras.scala

```diff
@@ -19,5 +23,41 @@ trait MappingExtras[F[_]] extends Mapping[F] {
         CursorField(field, _.field(underlyingField, None).flatMap(_.as[A].map(f)), List(underlyingField))
     }
   }
+      
+  /**
+    * A dispatching `TypeMapping` that selects a different underlying mapping based on the GraphQL
+    * `Type` and field name.
+    */
```

Comment:
This will be used for angles and wavelengths etc. that appear in multiple places?

Reply:
Yes, I think it's easier to deal with than `PrefixedMapping`. So we can go back and change those.